kubernetes/health.go

```diff
@@ -104,21 +104,8 @@ func GetHealth(in interface{}) HealthStatus {
 	}
 }
 
-func k8sHealth() cel.EnvOption {
-	return cel.Function("k8s.health",
-		cel.Overload("k8s.health_any",
-			[]*cel.Type{cel.AnyType},
-			cel.AnyType,
-			cel.UnaryBinding(func(obj ref.Val) ref.Val {
-				jsonObj, _ := conv.AnyToMapStringAny(GetHealth(obj.Value()))
-				return types.NewDynamicMap(types.DefaultTypeAdapter, jsonObj)
-			}),
-		),
-	)
-}
-
 func k8sGetHealth() cel.EnvOption {
-	return cel.Function("GetHealth",
+	return cel.Function("k8s.getHealth",
```

Comment:
I don't think we can just remove functions - we can mark them depreceated and dont document them, but we need to maintain backwards compatibility